registry/tests/admin_test.py

```diff
@@ -169,3 +170,42 @@ def testAdminListUserUI(self):
         assert user['packages'] == 1
         assert user['status'] == 'active'
         assert user['last_seen'] == '2018-01-14T19:33:27.656835Z'
+
+    def testAdminPackageUserUI(self):
+        self._mock_admin()
+        QUILT_AUTH_URL = quilt_server.app.config['QUILT_AUTH_URL']
+        user_list_api = "%s/accounts/users" % QUILT_AUTH_URL
+        self.requests_mock.add(responses.GET, user_list_api, json.dumps({
+            'status': 200,
+            'count': 1,
+            'next': None,
+            'previous': None,
+            'results': [{
+                'username': self.user,
+                'id': 1,
+                'date_joined': '2018-01-14T19:33:27.656835Z',
+                'email': 'admin@quiltdata.io',
+                'is_staff': True,
+                'is_superuser': True,
+                'is_active': True,
+                'last_login': '2018-01-14T19:33:27.656835Z'
+            }]
+            }))
+        resp = self.app.get(
+            '/api/admin/package_summary',
+            headers={
+                'Authorization': self.user
+            }
+        )
+        data = json.loads(resp.data.decode('utf8'))
+        assert data['status'] == 200
+        package = data['packages']['{user}/{pkg}'.format(user=self.user, pkg=self.pkg)]
+        now = datetime.datetime.utcnow()
+        last_push = package['pushes']['latest']
+        then = datetime.datetime.strptime(last_push, "%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
Actually, we should be returning UNIX timestamps everywhere - just to avoid dealing with parsing, timezones, etc. (Not sure if you've already used dates in other endpoints? If so, you should change those.)

Search for _utc_datetime_to_ts in views.py for examples.

Reply:
I don't generate new timestamps, just parse the ones that come out of the database.

Reply:
Yeah. But you should return them in the UNIX timestamp format.

Reply:
Done, also fixed for audit endpoints